src/components/presentational/SearchAndFilterInput.jsx

```diff
@@ -144,6 +147,91 @@ export default function SearchAndFilterInput(props) {
     props.setFilterString(myFilterString);
   }
 
+  // initialize pyGeoAPI flag
+  let pyGeoAPIFlag = false;
+
+  // New state for queryable titles
+  const [queryableTitles, setQueryableTitles] = useState([]);
+
+  // all collections
+  const collection = props.target.collections;
+
+  // retrieves all PyGEO collections
+  const isInPyAPI = collection.filter(data => data.hasOwnProperty('itemType'));
+
+  // finds and assigns the selected collection from the PYGEO api
+  const selectedCollection = isInPyAPI.find(data => data.title === props.selectedTitle);
+
+  // retrieves all pyGEO titles
+  const collectionTitles = isInPyAPI.map(data => data.title);
+
+
+
+  // checks if correct title selected
+  if (collectionTitles.includes(props.selectedTitle))
+  {
+    //set pyGeoAPI flag
+    pyGeoAPIFlag = true;
+
+    // set the selected link
+    let QueryableDirectoryLink = selectedCollection.links.find(link => link.rel === "queryables").href;
+
+    // creates URL to get the properties
+    let QueryableURL = 'https://astrogeology.usgs.gov/pygeoapi/' + QueryableDirectoryLink;
+
+    // fetches URL to get the properties
+    fetch(QueryableURL)
```

Comment:
This fetch gets hit continuously.

Reply:
Ill take a look at it but for some reason i cant find the error, do you mind sending over a screenshot of issue